examples/gnrc_border_router/Makefile

```diff
@@ -105,19 +80,34 @@ SHOULD_RUN_KCONFIG ?=
 
 include $(RIOTBASE)/Makefile.include
 
-ifndef CONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE
-ifeq (1,$(STATIC_ROUTES))
-  CFLAGS += -DCONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE=1
-  # CONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE=1 requires one more address for
-  # `fe80::2`.
-  CFLAGS += -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=3
-endif
+# Compile-time configuration for DHCPv6 client (needs to come after
+# Makefile.include as this might come from Kconfig)
+ifeq (1,$(USE_DHCPV6))
+  ifndef CONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE
+    ifeq (1,$(STATIC_ROUTES))
+      CFLAGS += -DCONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE=1
+      # CONFIG_GNRC_DHCPV6_CLIENT_6LBR_STATIC_ROUTE=1 requires one more address
+      # for `fe80::2`.
+      CFLAGS += -DCONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF=3
+    endif
+  endif
 endif
 
+
 .PHONY: host-tools
 
 host-tools:
 	$(Q)env -u CC -u CFLAGS make -C $(RIOTTOOLS)
 
+# define native specific targets to only run UHCP daemon when required
+ifeq (,$(filter native,$(BOARD)))
```

Comment:
So this will only run when *not* on native - I think you can just drop 'native specific targets' and the comment will make more sense.

Reply:
Oops this is supposed to be a `ifneq`!